docs/cookbook/importing_settings_from_a_bundle.md

```diff
@@ -97,9 +97,9 @@ class AcmeExampleExtension extends Extension implements PrependExtensionInterfac
         // Loading your YAML file containing template rules
         $configFile = __DIR__ . '/../Resources/config/template_rules.yaml';
         $config = Yaml::parse( file_get_contents( $configFile ) );
-        // Explicitly prepend loaded configuration for "ezpublish" namespace.
-        // It will be placed under the "ezpublish" configuration key, like in ezplatform.yaml.
-        $container->prependExtensionConfig( 'ezpublish', $config );
+        // Explicitly prepend loaded configuration for "ezplatform" namespace.
+        // It will be placed under the "ezplatform" configuration key, like in ezplatform.yaml.
+        $container->prependExtensionConfig( 'ezplatform', $config );
```

Comment:
Note: due to technical limitations for this to work, developer needs to place his bundle before EzPlatformCoreBundle in `bundles.php`. Please check though if we're not saying something contradictory elsewhere ;)